src/d3a/models/strategy/external_strategies/storage.py

```diff
@@ -129,6 +129,15 @@ def _offer(self, payload):
             assert set(arguments.keys()) == {'price', 'energy', 'transaction_id'}
             arguments['seller'] = self.device.name
             arguments['seller_origin'] = self.device.name
+            if arguments['energy'] > self.state.energy_to_sell_dict[self.market.time_slot]:
+                "ESS energy limit for this market_slot has been surpassed."
```

Comment:
This is probably a remnant. I wonder why the checks dont fail.
Please remove this line. Thanks!

Reply:
This is not an illegal statement as far as I know. Python allows single variables as statements, even without assignment, that's why it does not fail. I agree to remove the line though